ONSdigital/sdx-transform-testform
# Hh xml

**Changes**

Endpoint and xml template added for HH questionnaires
**How to test**

Depends on https://github.com/ONSdigital/sdx-compose/pull/7

In console select survey '0.hh2016.json' and'Submit'. View the message via the 'sdx-testform' queue in RabbitMQ Management, and assert that it is returning the correct format - see 
[HHUnitData.txt](https://github.com/ONSdigital/sdx-transform-testform/files/396154/HHUnitData.txt)


**Who can test**

Anyone but @jjeane
